tests/fixture/tmpnet/network_config.go

```diff
@@ -39,9 +41,6 @@ func (n *Network) Write() error {
 	if err := n.writeGenesis(); err != nil {
```

Comment:
Genesis should not be wrote for fuji/mainnet

Reply:
If may be a problem after reading it again, for when some initialization logic checks genesis for not being null. Probably not the case atm but maybe better to avoid the future  case.

Reply:
Made https://github.com/ava-labs/avalanchego/pull/3868 to cover for this